client/COPY.json

```
@@ -377,6 +377,8 @@
   "CRP_LETTER_RESP_WINDOW_TASK_LABEL": "CAVC-Remand-Processed-Letter Response Window Task",
   "CRP_LETTER_RESP_WINDOW_TASK_DEFAULT_INSTRUCTIONS": "The response window task will return to the CAVC Teams queue after 90 days or when manually ended early.",
 
+  "CAVC_SEND_TO_TEAM_BLOCKING_DISTRIBUTION_DETAIL": "Create a %s assigned to the %s team. This will not block you from completing this task or sending the 90-day letter to the veteran.",
```

In the UI I found the wording "this task" confusing, suggestion:

```suggestion
  "CAVC_SEND_TO_TEAM_BLOCKING_DISTRIBUTION_DETAIL": "Create a %s assigned to the %s team. This will not block you from completing your task to send the 90-day letter to the veteran.",
```